### ISSUE TITLE
Add data analysis vignette

### ISSUE THREAD

**Issue by gaow** [stephenslab/susieR#22]
For starters I'll add a finemapping vignette using some simulated data . I'll also perhaps work with Kaiqian to adopt / formalize one of her prediction application to a polished vignette.

**Comment by stephens999** [comment]
the prediction application is a bit premature. there are issues with
library size that
we really have not addressed, partly as I am not sure exactly how to do it
(although i have ideas) and
partly because it seems premature to bother with it - I wanted to get a
"quick look"
at those data to see what happened.

**Comment by stephens999** [comment]
Thx. Quick questions.

Why do you estimate prior variance? I suggest fixing this (the default)
until we have more experience with ways to estimate. You may want to
suggest people use two different values (0.1 and 0.2?) and check results do
not change too much. We might want to think more about this.

Do all the software licenses allow you to distribute dockerized versions of
other methods?

In any case I am honestly not sure you want to encourage running all
methods as a way to analyze the data. You may get many questions about
differences in results that we do not want to deal with. Or they may only
focus on results that agree between methods, which may down weight the
benefits of the best methods!

Running all methods seems more useful for people who want to compare and
improve methods, less so for people who are convinced that susie is the way
to go and want to use susieR to analyze data.
So if remove this to a place connected with your paper results rather than
this package.

Matthew

**Comment by gaow** [comment]
> Why do you estimate prior variance?

Okay i might have misunderstood one of your earlier comment we when talked in person. I'll change it as you suggested for now and we can talk about it again next time.

> Do all the software licenses allow you to distribute dockerized versions of
other methods?

Good question and apparently the answer is no for FINEMAP ... I've deleted it from dockerhub as we speak.

But I'll provide `Dockerfile` somewhere (maybe we have a `susie-paper` repository?) and users should be able to build the docker image with one line of command (plus >5min wait) if they have hard time configuring their computer to reproduce results for our paper. Does it sound reasonable?

> In any case I am honestly not sure you want to encourage running all
methods as a way to analyze the data. 

I was not encouraging it, but just wanted to point out similarities and differences results are summarized by each of these methods -- but I do agree that this is job for another place and when I have it in this repo I did implicitly encouraged people to use all methods. I'll move it elsewhere.
